pipelines/common/src/main/java/org/openmrs/analytics/DwhFiles.java

```diff
@@ -42,54 +55,36 @@ public class DwhFiles {
 
   private static final Logger log = LoggerFactory.getLogger(DwhFiles.class);
 
-  private static final String TIMESTAMP_FILE = "timestamp.txt";
+  public static final String TIMESTAMP_FILE = "timestamp.txt";
 
-  private static final String INCREMENTAL_DIR = "incremental_run";
+  public static final String INCREMENTAL_DIR = "incremental_run";
 
   private final String dwhRoot;
 
-  private final FileSystem fileSystem;
-
   private final FhirContext fhirContext;
 
   private DwhFiles(String dwhRoot) {
-    this(dwhRoot, FileSystems.getDefault(), FhirContext.forR4Cached());
-  }
-
-  static DwhFiles forRoot(String dwhRoot) {
-    return new DwhFiles(dwhRoot);
+    this(dwhRoot, FhirContext.forR4Cached());
   }
 
-  DwhFiles(String dwhRoot, FileSystem fileSystem, FhirContext fhirContext) {
+  DwhFiles(String dwhRoot, FhirContext fhirContext) {
     Preconditions.checkNotNull(dwhRoot);
     this.dwhRoot = dwhRoot;
-    this.fileSystem = fileSystem;
     this.fhirContext = fhirContext;
   }
 
-  public String getRoot() {
-    return dwhRoot;
+  static DwhFiles forRoot(String dwhRoot) {
+    return new DwhFiles(dwhRoot);
   }
 
-  /**
-   * This returns the path to the directory which contains resources of a specific type. It will
-   * also make sure the directory exists.
-   *
-   * @param resourceType the type of the FHIR resources
-   * @return The path to the resource type dir.
-   * @throws IOException if directory creation fails (won't fail if dir already exists).
-   */
-  public Path createResourcePath(String resourceType) throws IOException {
-    // TODO Check this code on GCS and make it file-system agnostic:
-    //   https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/issues/390
-    Path path = getResourcePath(resourceType);
-    Files.createDirectories(path);
-    return path;
+  public String getRoot() {
+    return dwhRoot;
   }
 
   /** Similar to {@code createResourcePath} but does not create the directory */
```

Comment:
nit: Please update the javadoc (remove reference to `createResourcePath`).

Reply:
Done